documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+```
+### Get Leaderboard by name
+You can use the ***Nether.Leaderboard.GetLeaderboardFromName*** method to get all of the gammer tags and highscores for the specified Leaderboard. Returns a LeaderboardSummary object
+#### Example usage
+```javascript
+var leaderboardScores = Nether.Leaderboard.GetLeaderboard('testboard');
+```
+
+### Put score
+You can use the ***Nether.Score.PutScore*** method to add a players score.
+#### Example usage
+```javascript
+Nether.Scores.AddScore('UK', 100, 'customTag');
+Nether.Scores.AddScore('UK', 100);
+```
+
+### Drop scores
+You can use the ***Nether.Score.DropMyScore*** method to delete all scores for the logged in player
+#### Example usage
+```javascript
+Nether.Scores.DropMyScore();
+```
+### Player Management
+
+### Get the current Player
+You can use the ***Nether.Player.GetCurrentPlayer*** method to get the current player details. Returns a Player object
+#### Example usage
+```javascript
+var player = Nether.Player.GetCurrentPlayer();
+```
+### Put Player
+You can use the ***Nether.Player.PutCurrentPlayer*** method to update Players details
+#### Example usage
+```javascript
+Nether.Player.PutCurrentPlayer('gammerTag', 'country', 'customTag');
+```
+### Add current player to group
+You can use the ***Nether.Player.AddCurrentPlayerToGroup*** method to add the current player to a group
+#### Example usage
+```javascript
+Nether.Player.AddCurrentPlayerToGroup('groupName');
+```
+
+### Delete current player
+You can use the ***Nether.Player.DeleteCurrentPlayer*** method to delete the current player
+#### Example usage
+```javascript
+Nether.Player.DeleteCurrentPlayer();
+```
+
+### Get current player state
+You can use the ***Nether.Player.GetCurrentPlayerState*** method to get the state of the current player. Returns a PlayerState object
+#### Example usage
+```javascript
+var playerState = Nether.Player.GetCurrentPlayerState();
+```
+
+### Put current player state
+You can use the ***Nether.Player.PutCurrentPlayerState*** method to update the current players player state.
+#### Example usage
+```javascript
+Nether.Player.PutCurrentPlayerState('gamerTag', 'state');
+```
+
+### Get Player Groups
+You can use the ***Nether.Player.GetPlayerGroups*** method to get the groups for the current player. Returns an array of Groups
+#### Example usage
+```javascript
+var groups = Nether.Player.GetPlayerGroups();
+```
+
+## Player Identity
+
+#### Get Gamer Tag From Player Id
+You can use the ***Nether.Player.Identity.GetGammerTag*** method to get the players gammer tag from their player Id. Returns a string with the gamer tag
+#### Example usage
+```javascript
+var gamerTag = Nether.Player.Identity.GetGammerTag('playerId');
+```
+
+#### Set Gamer Tag From Player Id
+You can use the ***Nether.Player.Identity.SetGammerTag*** method to set the players gammer tag
+#### Example usage
+```javascript
+Nether.Player.Identity.SetGammerTag('playerid', 'gammerTag');
+```
+## Administrate players
+
+### Get Players
```

Comment:
For the client SDK, we're focussing on the player functionality so let's remove the admin functionality to keep the scope down initially